### ISSUE TITLE
An error "inline post not available on this machine" happened during the task "gfsfcst" when running global-workflow with ufs-p8.

### ISSUE THREAD

**Issue by qy2ua** [ufs-community/ufs-weather-model#2083]
## Description

An error "inline post not available on this machine" happened during the task "gfsfcst" when running global-workflow with UFS-P8.

### To Reproduce:

I am running global-workflow (hash: 9a538ee) with UFS-P8 (hash: 9ae617e) on the HPC of the University of Arizona. The compiler is Intel. 
The setup command is "./setup_expt.py forecast-only --app S2SW --pslot test_c384 --idate 2012070100 --edate 2012070106 --resdet 384 --gfs_cyc 1 ...". 
After executing the command "rocotorun -d test_c384.db -w test_c384.xml", the first two task "gfscoupled_ic" and "gfswaveinit" showed "SUCCEEDED". However, the third task "gfsfcst" failed. The log file shows "inline post not available on this machine". The details about the log information are as below.

## Additional context

It seems that the error "inline post not available on this machine" is from "global-workflow/sorc/ufs_model.fd/FV3/io/module_wrt_grid_comp.F90". See below:

if( wrt_int_state%write_dopost ) then
#ifdef NO_INLINE_POST
        rc = ESMF_RC_NOT_IMPL
        print *,'inline post not available on this machine'
        if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, &
          line=__LINE__, file=__FILE__)) return
#endif

## Output

From the log file "test_c384/logs/2012070100/gfsfcst.log":

(...)

  0: NOTE from PE     0: before create fcst grid

  0:  dateS=hours since 2012-07-01 00:00:00date_init=        2012           7

  0:            1           0           0           0

  0:  in fv_phys bundle,nbdl=           2

  0:  add 3D field to after nearest_stod, fld=cldfra

  0:  fcst_initialize total time:    12729.0631773272

  0:  af fcstCom FBCount=            3

  0:  af allco wrtComp,write_groups=           1

405:  inline post not available on this machine

406:  inline post not available on this machine

407:  inline post not available on this machine

(...)

386: Abort(1) on node 386 (rank 386 in comm 496): application called MPI_Abort(comm=0x84000002, 1) - process 386

406: Abort(1) on node 406 (rank 406 in comm 496): application called MPI_Abort(comm=0x84000002, 1) - process 406

394: Abort(1) on node 394 (rank 394 in comm 496): application called MPI_Abort(comm=0x84000002, 1) - process 394

(...)

srun: error: r2u11n2: tasks 384-385,387-388,391: Exited with exit code 1

srun: error: r2u12n1: task 409: Exited with exit code 1

srun: error: r2u11n2: task 381: Exited with exit code 1

(...)

**Comment by DeniseWorthen** [comment]
@qy2ua The machine you are running on is not a Tier-1 Platform. Please see https://github.com/ufs-community/ufs/wiki/Supported-Platforms-and-Compilers

**Comment by qy2ua** [comment]
@DeniseWorthen Yes. But I have successfully installed global-workflow on my machine.

**Comment by gspetro-NOAA** [comment]
@qy2ua It likely still won't work without several tweaks if you're not running on a Tier-1 System. It sounds like the task you are trying to run uses inline post. You would likely have to dig into the source code and alter it to work for your system, but since your system isn't supported and we don't have access to it, there's very little we can do to help. You would probably have to try reconfiguring the task to run inline post or offline post (post-processing separately with the [UPP](https://github.com/NOAA-EMC/UPP)). Let us know if you still need assistance or if we can close this issue.

**Comment by gspetro-NOAA** [comment]
This issue has been inactive for over a year and relates to running on unsupported systems. If further assistance is required, the author may post a question in the [GitHub Discussions Q&A](https://github.com/ufs-community/ufs-weather-model/discussions/categories/q-a).
